aim2dat/strct/strct.py

```diff
@@ -562,7 +562,7 @@ def from_file(cls, file_path: str, attributes: dict = None, label: str = None) -
 
         Returns
         -------
-        aiida_scripst.strct.Structure
+        aim2dat.strct.strct.Structure
```

Comment:
We should best have in the documentation the highest-level import (this way we don't break the interface if we change the actual location of the class at some point...)
```suggestion
        aim2dat.strct.Structure
```